_opensearch/cluster.md

```diff
@@ -30,6 +30,7 @@ Cluster manager eligible | Elects one node among them as the cluster manager nod
 Data | Stores and searches data. Performs all data-related operations (indexing, searching, aggregating) on local shards. These are the worker nodes of your cluster and need more disk space than any other node type. | As you add data nodes, keep them balanced between zones. For example, if you have three zones, add data nodes in multiples of three, one for each zone. We recommend using storage and RAM-heavy nodes.
 Ingest | Pre-processes data before storing it in the cluster. Runs an ingest pipeline that transforms your data before adding it to an index. | If you plan to ingest a lot of data and run complex ingest pipelines, we recommend you use dedicated ingest nodes. You can also optionally offload your indexing from the data nodes so that your data nodes are used exclusively for searching and aggregating.
 Coordinating | Delegates client requests to the shards on the data nodes, collects and aggregates the results into one final result, and sends this result back to the client. | A couple of dedicated coordinating-only nodes is appropriate to prevent bottlenecks for search-heavy workloads. We recommend using CPUs with as many cores as you can.
+Dynamic | Delegates a specific node for custom work, such as Machine Learning (ML) tasks, preventing the consumption of resources from data nodes, therefore not affecting any OpenSearch functionality. 
```

Comment:
Change "Machine Learning (ML)" to "machine learning (ML)". Let's replace the last comma with "and" ("...from data nodes and therefore...).